### PR TITLE
Fix:SPCTR-2713: Adding eslintrc text domain rules

### PR DESCRIPTION
**Pull request by adi3890**
### Description
Adding rule 
`@wordpress/i18n-text-domain': [ 'error', { allowedTextDomain: 'ultimate-addons-for-gutenberg' }, ],`
in .eslintrc to fix all missing and wrong text-domains in js files

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

File: src/blocks/post/post-excerpt/save.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export default function save() {
 	return (
 		<div className="uagb-post__excerpt uagb-full_post">
 			{ __(
-				"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book."
+				"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.", 'ultimate-addons-for-gutenberg'
```
Comment (bsf-zanev) [inline src/blocks/post/post-excerpt/save.js:6]: Could we have a single quote string over here for the text?

**Review by bsf-zanev — APPROVED** [review]
Just added one comment below.

File: src/blocks/post/post-excerpt/save.js (hunk 1 above)
Comment (adi3890) [reply src/blocks/post/post-excerpt/save.js]: updated

File: src/blocks/post/post-title/save.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ export default function save() {
 		<h3 className="uagb-post__text uagb-post__title">
 			<a href="/" target="_blank" rel="noopener noreferrer" alt="">
 				{ ' ' }
-				{ __( 'Wordpress Post Title' ) }
+				{ __( 'Wordpress Post Title', 'ultimate-addons-for-gutenberg' ) }
```
Comment (patilvikasj) [inline src/blocks/post/post-title/save.js:7]:
```suggestion
				{ __( 'WordPress Post Title', 'ultimate-addons-for-gutenberg' ) }
```

Comment (adi3890) [reply src/blocks/post/post-title/save.js]: updated as suggested

**Review by patilvikasj — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**